packages/ui/src/Popup/AuthManagement/index.tsx

```diff
@@ -32,41 +32,40 @@ export function AuthManagement (): JSX.Element {
   return (
     <Flex flexDirection='column'
       height='100%'>
-      <Header headerText='Manage Website Access'
+      <Header headerText='Manage connected dApps'
         iconAsset={SvgFileLockOutline}
         width='100%'>
         <Box mb='m'
           mt='s'>
           <Text color='gray.0'
             variant='b2'>
-            Grant or deny these applications access to Polymesh Wallet.
+            Allow or deny these applications to connect to your Polymesh Wallet.
           </Text>
         </Box>
       </Header>
       {hasAuthList && (
         <Box p='s'
           width='100%'>
-          <Text color='gray.1'
-            variant='b2m'>
-            Filter by host name:
-          </Text>
           <InputFilter onChange={_onChangeFilter}
-            placeholder={'example.com'}
+            placeholder={'Search by website name...'}
             value={filter} />
         </Box>
       )}
-      <Box p='s'
-        style={{ overflowY: 'auto' }}
```

Comment:
Noticed the wasn't scrolling, looks like it got deleted accidentally

Reply:
👍